lesson07/assignment/hello-world/src/components/HistoryPage.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div>
+    <header>
+      <h1>Coffee History</h1>
+      <img src="/src/images/top-view-tasty-coffee-with-coffee-beans.jpg" alt="top view coffee with coffee beans" class="img-fluid">
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
```

Comment:
Don't need to include the navbar on each page.  In an ideal world the navbar is a component that is imported into your App.vue file.  Just having the navbar in the App.vue file is fine for this assignment.